app/models/site_config.rb

```diff
@@ -11,28 +11,23 @@ class SiteConfig < RailsSettings::Base
   # the cache, or call SiteConfig.clear_cache
   cache_prefix { "v1" }
 
+  # Authentication
```

Comment:
Simply Grouped this file better and mocked the structure in an alphabetised section manner that maps to the view for readability and ease of use